packages/react-animation/src/carousel/hooks/useCarouselControls.ts

```diff
@@ -0,0 +1,121 @@
+import { useRefValue } from '@mediamonks/react-hooks';
+import gsap from 'gsap';
+import { useMemo } from 'react';
+import { CarouselType, type CarouselContext } from './useCarousel.js';
+import { useCarouselCalculations } from './useCarouselCalculations.js';
+import { useCarouselIndex } from './useCarouselIndex.js';
+
+export type CarouselControls = {
+  next(): Promise<void>;
+  previous(): Promise<void>;
+  index(index: number): Promise<void>;
```

Comment:
I've added the option to set initial index to Foundation. See comments below for the code I've added.
Still behaves a bit buggy:

`setInitial(index: number): Promise<void>;`